CASMHMS-6589: Added extra verbiage to describe accuracy of CPU/GPU hardware history (#6199)
Cray-HPE/docs-csm: operations/hardware_state_manager/Remove_Duplicate_Detected_Events_From_HSM_Postgres_Database.md

> The steps in this document must be also be completed immediately prior to

Reviewer: If this is the case, would it make sense to call this out in `upgrade/Prepare_for_Upgrade_to_Next_CSM_Major_Version.md` ?

Author: Yes, an entirely new section was added to that document for this.

Reviewer: Excellent.

> starting a CSM 1.6.0 upgrade.

Reviewer: I assume this would also be true when upgrading to CSM 1.6.1 from CSM 1.5.
Would it apply also in an upgrade from CSM 1.6.0 to CSM 1.6.1?

```suggestion
starting an upgrade from CSM 1.5 to CSM 1.6.
```

Author: Thanks!  Change made.

> Unfortunately, due to the nature of this bug and capabilities of the pruning
> script, the history of FRUIDs associated to CPUs and GPUs prior to the CSM
> 1.7.0 release may not be fully accurate.  Once your system is upgraded to CSM
> 1.7.0, the accuracy of FRUIDs associated with CPUs and GPUs will be fully
> restored from that point in time and into the future.  No other component
> types were affected by this bug.

Reviewer: ```suggestion
Unfortunately, because to the nature of this bug and the capabilities of the pruning
script, the history of FRUIDs associated to CPUs and GPUs prior to the CSM
1.7.0 release may not be fully accurate. After the system is upgraded to CSM
1.7, the FRUIDs associations with CPUs and GPUs will be fully accurate from that point on.
No other component types were affected by this bug.
```

Author: FYI, I made these changes plus some additional changes in the area.
